app/src/DesignSystem/DirectoryInput.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import { MAIN_IPC_CHANNEL } from "../types.js";
+  import Button from "./Button.svelte";
+  import Input from "./Input.svelte";
+  import Text from "./Text";
+
+  export let placeholder = "";
+  export let style = null;
+  export let path = "";
+
+  let files;
+
+  const openFileDialog = async () => {
+    path = await window.electron.ipcRenderer.invoke(MAIN_IPC_CHANNEL);
```

Comment:
Can you add a comment to documentation on how this works? As I remember the normal file dialog not being sufficient.